lib/widgets/mxc_image.dart

```diff
@@ -7,6 +7,8 @@ import 'package:fluffychat/config/themes.dart';
 import 'package:fluffychat/utils/matrix_sdk_extensions/matrix_file_extension.dart';
 import 'package:fluffychat/widgets/matrix.dart';
 
+typedef MxcImageCacheMap = Map<String, Uint8List>;
```

Comment:
For me, hard to understand what is `Key`, what is `Value`, why not `typedef` for it? And Map should be Map.

Reply:
done